### ISSUE TITLE
[Suggestion] Ability to merge MPC projects.

### ISSUE THREAD

**Issue by MagmaSlime123** [CraftedMods/music-pack-creator#45]
Since no suggestion template exists on here nor on Discord, then I'll just
post it here. Allow multiple music pack projects to be merged into 1.

**Comment by CraftedMods** [comment]
Good idea. It will probably be a while until a next update is released, though.

**Comment by MagmaBro123** [comment]
Alright.
